prow/prow_secrets.md

```diff
@@ -41,7 +41,13 @@ where the prow service/build clusters are located.
    The above command ensures that the service account used by prow can only
    access the secret name `<my-gsm-secret-name>` in the GCP project owned by
    clients. The service account used for prow.k8s.io is defined in
```

Comment:
Please clarify that this first pair of serviceaccount+externalsecret is for the service cluster and anything running in the `test-infra-trusted` build cluster. It would also be good to clarify that the `k8s-prow-builds` cluster is the default build cluster.

Reply:
done